src/views/Dashboard.vue

```diff
@@ -219,7 +271,14 @@ export default {
     ProprietorPartnersListSm,
     StaffNotation,
     TodoList
-  },
+  }, // eslint-disable-line vue/multi-word-component-names
```

Comment:
Should this be a global rule override instead?

Reply:
This rule only applies for this file so I opted to have it only in this file. However, I could add it to the eslint config file as a global rule. Your call Sev!

Reply:
Weird that this rule only applies to Options API components...?

However, I think that, globally, we do want to allow multi-word component names.

Reply:
Oh, looks like this rule was already off ...?

Reply:
Yup, not sure why I added it in the first place 🤣 Must be my brain imagining things.